python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -473,6 +473,26 @@ def forward(self, x):
 def ElementwiseLeakyReluStaticModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(4, 5, 6, low=-1))
 
+# ==============================================================================
+
+
+class ElementwiseEluModule(torch.nn.Module):
```

Comment:
Also, add a test with non-default values of alpha, scale, and input_scale.

Reply:
Done